server/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -363,6 +363,66 @@ public void validate(IndexMode value, Map<Setting<?>, Object> settings) {
         Property.Final
     );
 
+    /**
+     * in time series mode, the start time of the index, timestamp must larger than start_time
+     * format is epoch_millis
+     */
+    public static final Setting<Long> TIME_SERIES_START_TIME = Setting.longSetting(
```

Comment:
I think it'd be a nice follow up change to make a `Setting<Date>` that took date in `DateFieldMapper.DEFAULT_DATE_TIME_FORMATTER`. Just ISO8601 and millis since epoch, basically. Maybe a little kinder to work with. That shouldn't be a "for now" thing though.